scripts/us_bea/states_gdp/import_data.py

```diff
@@ -0,0 +1,134 @@
+ # Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Pulls data from the US Bureau of Economic Analysis (BEA) on quarterly GDP
+per US state. Saves output as a CSV file.
+
+    Typical usage:
+
+    python3 import_data.py
+"""
+from urllib.request import urlopen
+from absl import app
+import pandas as pd
+import io
+import zipfile
+import csv
+import re
+
+# Suppress annoying pandas DF copy warnings.
+pd.options.mode.chained_assignment = None # default='warn'
+
+US_STATES = ['Alabama', 'Alaska', 'Arizona', 'Arkansas',
```

Comment:
Make this private: _US_STATES.  This means only code within this file (or perhaps code inheriting from the file?) can use it.  We do not want random code using this value and thus depending on this implementation detail.  Q.v., http://google.github.io/styleguide/pyguide.html#3162-naming-conventions.

Be sure to change all uses.